<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>100</x>
    <y>100</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion de Contacts</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nomLabel">
        <property name="text">
         <string>Nom :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nomLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="telephoneLabel">
        <property name="text">
         <string>Téléphone :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="telephoneLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="emailLabel">
        <property name="text">
         <string>Email :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="emailLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="adresseLabel">
        <property name="text">
         <string>Adresse :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="adresseLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="contactsTable">
      <property name="columnCount">
       <number>4</number>
      </property>
      <property name="horizontalHeaderLabels">
       <stringlist>
        <string>Nom</string>
        <string>Téléphone</string>
        <string>Email</string>
        <string>Adresse</string>
       </stringlist>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="ajouterButton">
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="modifierButton">
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="supprimerButton">
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rechercherButton">
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
